app/src/test/java/edu/uc/forbesne/shoppingsidekick/ProductDataUnitTest.kt

```diff
@@ -62,14 +62,14 @@ class ProductDataUnitTest {
         allProducts.add(banana)
 
         allProductsLiveData.postValue(allProducts)
-        every { productService.fetchProductsByName(or("Apple", "Banana")) } returns allProductsLiveData
-        every { productService.fetchProductsByName(not(or("Apple", "Banana"))) } returns MutableLiveData<ArrayList<Product>>()
+        every { productService.fetchProductsByDescription(or("Apple", "Banana")) } returns allProductsLiveData
+        every { productService.fetchProductsByDescription(not(or("Apple", "Banana"))) } returns MutableLiveData<ArrayList<Product>>()
```

Comment:
I'd consider "Apple" and "Banana" a name, not a description.   I'd keep this as is.